spec/knapsack_pro/test_file_finder_spec.rb

```diff
@@ -154,5 +154,23 @@
         end
       end
     end
+
+    context 'when KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE is defined' do
+      let(:test_file_list_source_file) { 'spec_fake/fake_test' }
```

Comment:
We could put this fake test file (with list of test files) in fixtures directory: `spec/fixtures/test_file_list_source_file.txt`